plugins/module_utils/hmc_resource.py

```diff
@@ -491,6 +491,24 @@ def installOSFromNIM(self, loc_code, nimIP, gateway, lparIP, vlanID, vlanPrio, s
             " " + viosName + " " + profName + " " + systemName
         self.hmcconn.execute(lpar_netboot)
 
+    def installlinuxOSFromNIM(self, nimIP, gateway, lparIP, submask, viosName, profName, systemName, lparMac):
```

Comment:
Don't Introduce new method, modify the existing method **installOSFromNIM**

Reply:
modified the code using existing method